api/src/main.rs

```diff
@@ -65,14 +65,17 @@ async fn async_main() -> anyhow::Result<()> {
 fn init_sentry() -> anyhow::Result<Option<sentry::ClientInitGuard>> {
     if let Ok(config) = load_config::<ApiConfig>() {
         if let Some(sentry_config) = &config.sentry {
-            info!("Initializing Sentry with DSN");
+            info!("initializing sentry with supplied dsn");
 
             let environment = Environment::load()?;
             let guard = sentry::init(sentry::ClientOptions {
                 dsn: Some(sentry_config.dsn.parse()?),
                 environment: Some(environment.to_string().into()),
                 traces_sample_rate: 1.0,
                 max_request_body_size: sentry::MaxRequestBodySize::Always,
+                integrations: vec![Arc::new(
+                    sentry::integrations::panic::PanicIntegration::new(),
+                )],
```

Comment:
We have our own custom panic hook here: https://github.com/supabase/etl/blob/1d8fd7a3fa22cefc38fb0007c65768fe244719a2/telemetry/src/lib.rs#L178-L209. Will this panic integration call our panic hook as well?

Reply:
Checked

```rust
fn setup(&self, _cfg: &mut ClientOptions) {
        INIT.call_once(|| {
            let next = panic::take_hook();
            panic::set_hook(Box::new(move |info| {
                panic_handler(info);
                next(info);
            }));
        });
    }
```

It does chain panic hooks.